packages/babel-generator/src/printer.ts

```diff
@@ -1036,6 +1036,15 @@ class Printer {
 
     let val;
     if (isBlockComment) {
+      const { _parenPushNewlineState } = this;
+      if (
+        _parenPushNewlineState?.printed === false &&
+        lastCharCode != charCodes.lineFeed &&
```

Comment:
Is there a case in which `_parenPushNewlineState?.printed` is false, `lastCharCode` _is_ `charCodes.lineFeed` and we do not need to print the parenthesis?

Reply:
Sorry I didn't understand what you meant, if `lastCharCode` is `charCodes.lineFeed` then it will not continue.

Reply:
Sorry, I meant: if we deleted the `lastCharCode != charCodes.lineFeed &&` check, there would be any difference?

Reply:
Oh, thank you! There is no difference between them. When `lastCharCode` is `charCodes.lineFeed`, `_parenPushNewlineState?.printed` must be `true`